auslib/blobs/apprelease.py

```diff
@@ -585,6 +617,23 @@ def createSnippets(self, updateQuery, update_type, whitelistedDomains, specialFo
         # We have no tests that require this, probably not worthwhile to implement.
         return {}
 
+    def getReferencedReleases(self):
+        """
+        :return: Returns set of names of partially referenced releases that the current
+        release references
+        """
+        referencedReleases = set()
+        if self.get('platforms'):
+            for platform in self['platforms']:
+                if self['platforms'][platform].get('locales'):
+                    for locale in self['platforms'][platform]['locales']:
+                        if self['platforms'][platform]['locales'][locale].get('partials'):
+                            for partials in self['platforms'][platform]['locales'][locale]['partials']:
```

Comment:
Nit: "partial" instead of "partials" as the variable name here - the loop only works on one at a time.